bin/git-fork

```diff
@@ -54,9 +87,30 @@ if [ "$origin" = true ]; then
     git fetch origin
 else
     # clone forked repo into current dir
-    git clone "${remote_prefix}${user}/${project}.git" "$project"
-    # add reference to origin fork so can merge in upstream changes
-    cd "$project"
-    git remote add upstream "${remote_prefix}${owner}/${project}.git"
-    git fetch upstream
+    git clone "${remote_prefix}${user}/${project}.git" "$project" 2>/dev/null
+
+    # Check to make sure we cloned the repo. Backoff exponetially and try again
+    timeout=( 2 2 2 6 6 6 20 20 )
+    total_wait=0
+    until [[ (( "$total_wait" > 60 )) || -d "$project" ]]; do
+      echo "Github is being slow today. Waiting $timeout secs to attempt clone."
+      sleep ${timeout[0]}
+      total_wait=$(expr $total_wait + ${timeout[0]})
```

Comment:
I thought about that, but some of the older shells don't understand the double parenthesis. So I opted for the more traditional form.